webapp/users.py

```diff
@@ -18,32 +34,28 @@ def __init__(self, name, link='/remote'):
         self.name = name
         self.link = link
 
-class User(UserMixin):
-    """A class for instantiating saved user accounts.
-        There should be 1 object of this class type per user account.
-
-    """
-    def __init__(self, name):
-        self._id = name
-        self._remotes = []
-        self._config = {}
-        self._load_config()
-
+class User(db.Model):
+    __tablename__ = 'users'
+    id = db.Column('user_id',db.Integer, primary_key=True)
+    username = db.Column('username', db.String(20), unique=True, index=True)
+    password = db.Column('password', db.String(10))
```

Comment:
We should be hashing the passwords for security sake. I'd recommend to look at [this article](https://pythonprogramming.net/password-hashing-flask-tutorial/) for more info.